content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,124 @@ draft: false
+**Presenter:** William Hearn (Senior Cloud Architect, GC Cloud One)  
+**Attendees:** ARC Members
+**Objective:** Review the Aurora initiative and secure approval for its **strategic direction and technical architecture**.
+
+---
+
+## 1. Executive Summary
+
+### **Context**
+
+The **Aurora Initiative** is a **unified application hosting platform** designed to standardize **managed Kubernetes cloud offerings** across SSC. It integrates **both proprietary (OpenShift) and OSS (Cloud Native Kubernetes) environments**, drawing from operational experience in the **Cloud Native Platform (CNP)**.
+
+### **Why Now?**
+
+- Current departmental cloud-native strategies are **disjointed, costly, and redundant**.
+- Aurora provides a **governance framework** that reduces inefficiencies, promotes **multi-cloud standardization**, and aligns with SSC’s **GC Cloud One** strategy.
+- The initiative has **received broad buy-in** from multiple departments and aligns with similar government initiatives (BC Gov DevOps, Platform One, etc.).
+
+---
+
+## 2. Architecture Overview
+
+### **2.1 Technical Architecture**
+
+- **Core Services:** Kubernetes (AKS, EKS, GKE, OpenShift)
+- **Networking:** Cilium + BGP (BIRD with route reflectors)
+- **Security Model:** Zero Trust, Gatekeeper (OPA), SCED for on-prem connectivity
+- **CI/CD:** GitLab Runners, Crossplane for cloud resource automation
+- **Observability:** Loki, Prometheus
+- **Multi-Tenancy Approach:** Namespaced isolation, policy enforcement
+
+#### **Key Benefits**
+
+✔️ **Unified API surface** for Kubernetes adoption across SSC.  
+✔️ **Flexible deployment models** (OSS and proprietary).  
+✔️ **Security-first** approach (zero trust, policy-driven enforcement).  
+✔️ **Interoperability** with existing departmental solutions.
+
+---
+
+## 3. Governance & Compliance Alignment
+
+- Aurora **adheres to SSC’s cloud governance model**, ensuring **interoperability, security, and standardization**.
+- Leverages **CNCF best practices** for operational maturity.
+- Supports **multi-tenancy, workload separation, and automated compliance checks**.
+- Aligns with **Treasury Board Secretariat (TBS) cloud directives**.
+
+### **Key Decision Point for ARC**
+
+✔️ Does Aurora meet SSC’s **interoperability and security requirements**?
+
+---
+
+## 4. Operational Model
+
+### **4.1 Deployment Approach**
+
+Aurora will be deployed in SSC’s **Enterprise Tenant**, with:
+
+- **Two main environments:** `AuroraNonProd` (mgmt-dev-cc-00) and `AuroraProd` (mgmt-prod-cc-00).
+- Integration with **Fortigate Firewalls** for automated rule management.
+- Centralized **GitOps pipeline** (Argo CD) for policy-based application deployment.
+
+### **4.2 Team Structure**
+
+- **Architecture & Governance Team**: Defines standards and compliance.
+- **Platform Operations Team**: Maintains core services (GitLab, Vault, Artifactory).
+- **Advisory SIGs (Architecture, Networking, Security)**: Drives technical decisions.
```

Comment:
```suggestion
- **Advisory SIGs (Architecture, Networking, Security)**: Drives technical decisions.
```

I think this will need to switch to TAG instead of SIG